### PR TITLE
[no ticket][risk=no] Puppeteer Combining two CDR modal tests

### PR DESCRIPTION
**Pull request by aweng98**
Combining two small CDR modal tests into one. Faster playback time, one less login and one less new workspace.

### CODE REVIEW

**Comment by aweng98** [comment]
@jmthibault79 Hi Joel, when you have time, can you take a look? Does change makes sense? Thanks

File: e2e/tests/workspace/cdr-version-upgrade.spec.ts [hunk 1]
```diff
@@ -1,69 +1,62 @@
...
-        const newVersionFlag = await workspacePage.getNewCdrVersionFlag();
-        await newVersionFlag.click();
+      const duplicationPage = new WorkspaceEditPage(page);
+      const upgradeMessage = await duplicationPage.getCdrVersionUpgradeMessage();
+      expect(upgradeMessage).toContain(workspaceName);
+      expect(upgradeMessage).toContain(`${config.altCdrVersionName} to ${config.defaultCdrVersionName}.`);
 
-        const modal = new CdrVersionUpgradeModal(page);
-        expect(await modal.isLoaded()).toBe(true);
+      cancelButton = await duplicationPage.getCancelButton();
```
Comment (jmthibault79) [inline e2e/tests/workspace/cdr-version-upgrade.spec.ts:43]: This is a different Cancel button from the button above, so I'd prefer a different variable name.  Then both of these can be const.

**Review by jmthibault79 — APPROVED** [review]
Looks good, thanks!

File: e2e/tests/workspace/cdr-version-upgrade.spec.ts (hunk 1 above)
Comment (aweng98) [reply e2e/tests/workspace/cdr-version-upgrade.spec.ts]: Done. Thanks.

**Pull request merged**